src/convert-old-lara/convert.ts

```diff
@@ -0,0 +1,391 @@
+  item.embeddable.choices.forEach((choice: Record<string, any>) => {
+    const convertedChoice: IChoice = {
+      id: `${choiceId}`,
+      content: choice.choice,
+      correct: choice.is_correct,
+      choiceFeedback: choice.prompt
+    };
+    convertedChoices.push(convertedChoice);
+    choiceId++;
+  });
+
+  const layout = item.embeddable.show_as_menu ? "Dropdown" : item.embeddable.layout;
+
+  const authoredState: IMultipleChoiceAuthoredState = {
+    version: 1,
+    questionType: "multiple_choice",
+    choices: convertedChoices,
+    layout
+  };
+
+  const authoredStateProperties: Record<string, string> = {
+    prompt: "prompt",
+    enable_check_answer: "enableCheckAnswer",
+    is_prediction: "required",
+    multi_answer: "multipleAnswer",
+    hint: "hint",
+    give_prediction_feedback: "customFeedback",
+    prediction_feedback: "predictionFeedback"
+  };
+
+  for (const key in authoredStateProperties) {
+    if (item.embeddable[key]) {
+      (authoredState as Record<string, any>)[authoredStateProperties[key]] = item.embeddable[key];
+    }
+  }
+
+  item.embeddable.authored_state = JSON.stringify(authoredState).replace(/"/g, '"');
+  item.embeddable.library_interactive = setLibraryInteractive(libraryInteractive);
+};
+
+const convertOpenResponse = (item: Record<string, any>, libraryInteractive: Record<string, any>) => {
+  const authoredState: IOpenResponseAuthoredState = {
+    version: 1,
+    prompt: item.embeddable.prompt,
+    defaultAnswer: item.embeddable.default_text,
+    required: item.embeddable.is_prediction,
+    questionType: "open_response",
+    predictionFeedback: item.embeddable.give_prediction_feedback ? item.embeddable.prediction_feedback : undefined,
+    hint: item.embeddable.hint
+  };
+
+  item.embeddable.authored_state = JSON.stringify(authoredState);
+  item.embeddable.library_interactive = setLibraryInteractive(libraryInteractive);
+};
+
+const convertImage = (item: Record<string, any>, libraryInteractive: Record<string, any>) => {
+  const authoredState: IImageAuthoredState = {
+    version: 1,
+    url: item.embeddable.url,
+    caption: item.embeddable.caption,
+    credit: item.embeddable.credit,
+    creditLink: item.embeddable.credit_url,
+    creditLinkDisplayText: "",
+    allowLightbox: true
+  };
+
+  item.embeddable.authored_state = JSON.stringify(authoredState);
+  item.embeddable.library_interactive = setLibraryInteractive(libraryInteractive);
+
+};
+
+const convertImageQuestion = (item: Record<string, any>, libraryInteractive: Record<string, any>) => {
+  const authoredState: IImageQuestionAuthoredState = {
+    version: 1,
+    questionType: "image_question",
+    required: item.embeddable.is_prediction,
+    predictionFeedback: item.embeddable.prediction_feedback,
+    backgroundImageUrl: item.embeddable.bg_url,
+    imageFit: "shrinkBackgroundToCanvas",
+    imagePosition: "center",
+    answerPrompt: item.embeddable.prompt,
+    prompt: item.embeddable.drawing_prompt,
```

Comment:
@emcelroy, I think `answerPrompt` and `prompt` should be assigned in the opposite way:
```
 answerPrompt: item.embeddable.drawing_prompt,
 prompt: item.embeddable.prompt,
```
I'm 90% sure, so you could double check in LARA.

Reply:
You're right. I kept getting those confused. That's fixed now.